landoui/auth.py

```diff
@@ -0,0 +1,79 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
If you run `invoke lint` you will see it wants some changes in the this file and some others.
`invoke format` can fix those too.